examples/tutorials/rest-api-annotated-service/build.gradle

```
@@ -0,0 +1,17 @@
+plugins {
+    id 'application'
+}
+
+dependencies {
+    implementation project(':core')
+    runtimeOnly 'org.slf4j:slf4j-simple'
+
+    testImplementation project(':junit5')
+    testImplementation 'net.javacrumbs.json-unit:json-unit-fluent'
+    testImplementation 'org.assertj:assertj-core'
+    testImplementation 'org.junit.jupiter:junit-jupiter-api'
+}
+
+application {
+    mainClassName = 'example.armeria.server.blog.Main'
```

`mainClassName` has been deprecated in Gradle 7.x.
```suggestion
    mainClass.set('example.armeria.server.blog.Main')
```